web/eslint.config.mjs

```diff
@@ -0,0 +1,24 @@
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import js from "@eslint/js";
+import { FlatCompat } from "@eslint/eslintrc";
+import { fixupConfigRules } from "@eslint/compat";
+import ts from "typescript-eslint";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+  recommendedConfig: js.configs.recommended,
+  allConfig: js.configs.all,
+});
+
+const patchedConfig = fixupConfigRules([...compat.extends("next/core-web-vitals")]);
+
+const config = [
+  ...patchedConfig,
+  ...ts.configs.recommended,
+  { ignores: [".next/*", "dist/*", "cypress/*", "cypress.config.ts"] },
+];
```

Comment:
It looks like this is ignoring the cypress folder, but then you have another eslint config inside the cypress folder. Is this intentional?

Reply:
yes, I thought we still wanted to lint cypress but just ignore some rules